<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="search_label">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>12</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>79</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>üîç</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_lineedit">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>80</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="filter_label">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filters</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="daycheckbox">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>164</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="monthcheckbox">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>211</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="yearcheckbox">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>260</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="price_checkbox">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>309</y>
      <width>128</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Price Limit Range</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="incomecheckbox">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>354</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Income</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="costcheckbox">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>400</y>
      <width>54</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Cost</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cost_lineedit">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>397</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter The Cost</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="day_lineedit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>161</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter The Day</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="month_lineedit">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>210</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter The Month</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="year_lineedit">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>260</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter The Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price_high">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>308</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter High Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="income_lineedit">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>350</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter The Income</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price_low">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>308</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Low Price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_btn">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>394</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Return MainPage</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
